src/averageAges.js

```diff
@@ -15,6 +15,16 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
+  const men = (century !== undefined)
+    ? people.filter(person => {
+      return person.sex === 'm'
+            && Math.ceil(person.died / 100) === century;
+    })
+    : people.filter(person => person.sex === 'm');
```

Comment:
```suggestion
  const men = people.filter(person =>
    century
      ? Math.ceil(person.died / 100) === century && person.sex === 'm'
      : (person.sex === 'm')
  );
```

You can actually simplify your code here and use .filter method only once.